changelogs/current.yaml

```diff
@@ -466,6 +466,10 @@ new_features:
     Added maximum gRPC message size that is allowed to be received in Envoy gRPC. If a message over this limit is received,
     the gRPC stream is terminated with the RESOURCE_EXHAUSTED error. This limit is applied to individual messages in the
     streaming response and not the total size of streaming response. Defaults to 0, which means unlimited.
+- area: load shed point
+  change: |
+    Added load shed point ``envoy.load_shed_points.http_downstream_filter_check`` that makes load shed check availabe in HTTP filters,
+    and right now it is avaiable in router. It will send local reply directly when Envoy is under pressure, typically memory.
```

Comment:
s/avaiable/available

Reply:
it's also only available for decoder filters

Reply:
Thanks! done